sources/packages/backend/apps/queue-consumers/src/processors/models/processors.models.ts

```diff
@@ -1,4 +1,5 @@
 import { LoggerService } from "@nestjs/common";
+import { ProcessSummary } from "@sims/utilities/logger";
 
 export interface QueueProcessSummaryResult {
```

Comment:
Moving forward as discussed errors and warnings would not be required in the `QueueProcessSummaryResult ` right?

Reply:
Up to the developer or scenario. I would say that would not be required in most of the cases.
I have no plans to change the `QueueProcessSummaryResult` right now.